README.md

```diff
@@ -401,7 +409,123 @@ via [additional environment variables](https://github.com/FIWARE/helm-charts/blo
+VerifiableCredentials for JWT, that can be used for authorization and authentication in down-stream components.
+* [Credentials Config Service](https://github.com/fiware/credentials-config-service) manages and provides information about services and the credentials they are using.
+It returns the scope to be requested from the wallet per service. Furthermore, it specifies the credentials required and
+the issuers list endpoints to validate against, when checking access for a certain service.
+* The [Keycloak-VC-Issuer](https://github.com/fiware/keycloak-vc-issuer) is plugin for Keycloak to support SIOP-2/OIDC4VP clients and issue VerifiableCredentials through
+the OIDC4VCI-Protocol to compliant wallets.
+* [PDP](https://github.com/fiware/dsba-pdp) is an implementation of a Policy-Decision Point, evaluating Json-Web-Tokens containing VerifiableCredentials in a
+DSBA-compliant way. It also supports the evaluation in the context of i4Trust.
+* [Keyrock](https://github.com/ging/fiware-idm) is the FIWARE component responsible for Identity Management. Using Keyrock (in conjunction with other security
+components) enables you to add OAuth2-based authentication and authorization security to your services and applications.
+* [Kong Plugins](https://github.com/fiware/kong-plugins-fiware) allow to extend the API Gateway Kong by further functionalities required for FIWARE-based environments. Kong
+Gateway is a lightweight, fast, and flexible cloud-native API gateway. An API gateway is a reverse proxy that lets you
+manage, configure, and route requests to your APIs.
+
+#### How to deploy
+
+The recommended way of deployment is via the provided [Helm charts](https://github.com/DOME-Marketplace/iam-components).
+
+To deploy a setup,
+the [umbrella chart](https://helm.sh/docs/howto/charts_tips_and_tricks/#complex-charts-with-many-dependencies) of the
+iam-components can be used as followed:
+
+- create a configuration values file according to the own environment, as described [here](#how-to-configure-1).
+- add helm chart repository to helm installation
+  ```
+    helm repo add dome-iam https://dome-marketplace.github.io/iam-components
+    helm repo update
+  ```
+  > :bulb: All releases of the IAM-components reside in the helm-repository https://dome-marketplace.github.io/iam-components.
+  In addition to that, all Pre-Release versions(build from the Pull Requests) are provided in the
+  pre-repo https://dome-marketplace.github.io/iam-components/pre. The pre-repo will be cleaned-up from time to time, in
+  order to keep the index manageable.
+
+- install the components using the prepared configuration
+  ```
+    helm install <RELEASE_NAME> dome-iam/iam-components --namespace <NAME_SPACE> --version <CHART_VERSION> -f values.yaml
+  ```
+
+#### How to configure
+
+Before deploying these components, you need to update some values:
```

Comment:
I would rephrase this a bit, so that people don't think that these are the only values to adapt. In the end, it is suggested to go through all parameters (the links to the correspondnig charts are provided below to look this up), but these parameters listed below are important to set and should be updated at least. Something like that.
Are we sure the deployment would work, when all other parameters are left with the default values?

Reply:
Not really, the accessible URLs still need to be replaced, and I didn't set any rbac and serviceAccount